Formula/now-cli.rb

```diff
@@ -0,0 +1,26 @@
+require "language/node"
+
+class NowCliBeta < Formula
```

Comment:
```suggestion
class NowCli < Formula
```

This isn't a beta and it doesn't match the naming convention.

Reply:
My bad.